narwhal/types/src/primary.rs

```diff
@@ -909,14 +909,6 @@ impl Certificate {
         }
     }
 
-    pub fn new_test_empty(protocol_config: &ProtocolConfig, author: AuthorityIdentifier) -> Self {
```

Comment:
Are these now unused, or should we just return V1 instead?

Reply:
They are unused in our codebase which is why I didn't get any errors that I was mistakenly returning CertV1